src/main/drivers/pwm_output_dshot.c

```diff
@@ -467,11 +467,9 @@ void pwmDshotMotorHardwareConfig(const timerHardware_t *timerHardware, uint8_t m
 #ifdef USE_DSHOT_DMAR
     if (useBurstDshot) {
         motor->timer->dmaBurstRef = dmaRef;
-
-        if (!configureTimer) {
-            motor->configured = true;
```

Comment:
what was `configureTimer` / `motor->configured` used for?

Reply:
Seems like `pwmWriteDshotInt` will return unless `motor->configured`.. https://github.com/betaflight/betaflight/blob/bbedb60fe20ea68f48a7a976168b5ea91d2eea5b/src/main/drivers/pwm_output_dshot.c#L82-L84

Reply:
This was used in order not to initialize a timer or burst dma stream multiple times when several channels of the same timer were used. But it wasn’t implemented consistently before (in some cases they were initialized multiple times anyway) and initializing multiple times doesn’t have negative consequences.
My code refactoring for bidir broke this code so I just eliminated the attempted avoidance of multiple initialization altogether to fix the associated bugs.

Reply:
Configured is set lower in the method, too, @fujin.